### DISCUSSION TITLE
[FEATURE-REQUEST] Iconos personalizados SVG

### DISCUSSION THREAD

**Discussion by facundolazcano** [Cotalker/documentation#205] [3. Ideas & Feature Requests] [answered]
Estimados, encontre una pequeño detalle. Uno intuitivamente cololca el PATH, pero la herramienta deberia especificar que es single PATH y el tamaño que necesita. En relación a esto último pensaba quizas hacer un resize.
Para introducir el Path y que funcionara utilice la siguiente libreria de python [](https://cairosvg.org/), y ocupe el siguiente comando: 

cairosvg football.svg --scale 0.1  -o futball_10.svg

Esto si bien trae otra complicación, el cual seria encontrar el factor de escalamiento segun el Path, y además ver si el algoritmo requiere el SVG completo. En ese sentido estoy abierto a seguir investigando para automatizar la forma en que se agregan iconos, incluso existen algunas herramientas nuevas para transformar imagenes a SVG.

**Comment by acedward** [comment, marked as answer]
@facundolazcano súper buena pregunta, y te tengo buenas noticias. Ahora estamos en capacidad de mejorar esto.
Sólo para comentarte que esto era necesario por el soporte limitado de SVGs en iOS y Android nativamente. Pero ahora con nuestra nueva aplicación - podemos volver a revisitar esto y soportar muchos más variedad de formatos.

¿Puedes solucionar tu problema actualmente o requieres algún tipo de soporte?
